frontend/src/components/DatabaseTable.vue

```diff
@@ -51,6 +51,17 @@
       <BBTableCell v-if="showProjectColumn" class="w-[15%]">
         <div class="flex flex-row space-x-2 items-center">
           <div>{{ projectName(database.project) }}</div>
+          <div
+            v-if="
+              mode === 'ALL_SHORT' && database.project.tenantMode === 'TENANT'
```

Comment:
I consider this icon could also be displayed under "ALL" | "ALL_SHORT" | "INSTANCE", wdyt?